fix/stuckdoors.lua

```diff
@@ -0,0 +1,120 @@
+-- Fix doors that are frozen in 'open' state.
+-- This may happen after people mess with the game by (incorrectly) teleporting units or items
+-- A door may stick open if the map occupancy flags are wrong.
+
+--[====[
+
+    fix/stuckdoors
+    ===============
+    Fix doors that are stuck open due to incorrect map occupancy flags, eg due to
+    incorrect use of `teleport`.
+
+]====]
```

Comment:
The docs have moved to `docs/fix/stuckdoors.rst`, but I haven't gone through and cleaned docstrings out of the script files yet. If you made any improvements to the help text, could you move the changes to the .rst file? Then you can remove this docstring entirely. It's not used by anything and will only get out of date.